docs/quickstart/install-and-use-a-package-in-visual-studio.md

```diff
@@ -1,78 +1,86 @@
 
-NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a Visual Studio project using the NuGet Package Manager, the [Package Manager Console](../consume-packages/install-use-packages-powershell.md), or the [dotnet CLI](install-and-use-a-package-using-the-dotnet-cli.md). This article demonstrates the process using the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package and a Windows Presentation Foundation (WPF) project. The same process applies to any other .NET or .NET Core project.
+*NuGet* packages contain Microsoft-supported reusable code that other developers have made available to you for use in your projects. You can install NuGet packages in a Microsoft Visual Studio project by using the NuGet Package Manager, the Package Manager Console, or the .NET CLI. This article demonstrates how to create a Windows Presentation Foundation (WPF) project with the popular `Newtonsoft.Json` package. The same process applies to any other .NET or .NET Core project.
 
-Once installed, refer to the package in code with `using <namespace>` where \<namespace\> is specific to the package you're using. Once the reference is made, you can call the package through its API.
+After you install a package, you can then make a reference to it in your code with the `using <namespace>` statement, where \<namespace\> is the name of package you're using. After you've made a reference, you can then call the package through its API.
 
-> [!Tip]
-> **Start with nuget.org**: Browsing *nuget.org* is how .NET developers typically find components they can reuse in their own applications. You can search *nuget.org* directly or find and install packages within Visual Studio as shown in this article. For general information, see [Find and evaluate NuGet packages](../consume-packages/finding-and-choosing-packages.md).
+The article is for Windows users only. If you're using Visual Studio for Mac, see [Install and use a package in Visual Studio for Mac](install-and-use-a-package-in-visual-studio-mac.md).
+
+> [!TIP]
+> To find a NuGet package, start with *nuget.org*. Browsing for nuget.org is how .NET developers typically find components they can reuse in their own applications. You can do a search of nuget.org directly or find and install packages within Visual Studio as shown in this article. For more information, see [Find and evaluate NuGet packages](../consume-packages/finding-and-choosing-packages.md).
 
 ## Prerequisites
 
-- Visual Studio 2019 with the .NET Desktop Development workload.
+- Install Visual Studio 2022 for Windows with the .NET desktop development workload.
+
+  You can install the 2022 Community edition for free from [visualstudio.microsoft.com](https://visualstudio.microsoft.com/), or use the Professional or Enterprise edition.
+
+## Create a WPF project
```

Comment:
I think it'd be great for these quick starts to use the modern project templates, such as ones that are .NET SDK based, https://github.com/NuGet/docs.microsoft.com-nuget/issues/2853.

I'm not asking/suggesting a change here, mostly just linking the issues.

Reply:
It looks like you can create a WPF template for .NET instead of .NET Framework. It would be an improvement to use the newer templates.

Reply:
Please use **Create a project** for the heading instead of **Create a WPF project**. We want to avoid implying that a WPF project is a requirement rather than just an example.

Reply:
For the WPF template for .NET (which I think is WPF Application - A project for creating a .NET Application), should I tell the user to select the .NET Core 3.1 or .NET 6.0 framework during the install?